test/index.js

```diff
@@ -27,11 +27,11 @@ test.before(async () => {
     path: pathToBinary
   });
 
-  await app.start();
+  return app.start();
```

Comment:
Would be better to use a ESLint ignore comment until it's fixed: https://github.com/avajs/eslint-plugin-ava/issues/147 I would also add a `TODO` comment about removing the ignore when fixed in eslint-plugin-ava.